x-pack/plugins/secops/server/lib/overview/query.dsl.ts

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { createQueryFilterClauses } from '../../utils/build_query';
+import { RequestBasicOptions } from '../framework';
+
+export const buildOverviewNetworkQuery = ({
+  filterQuery,
+  timerange: { from, to },
+  sourceConfiguration: {
+    fields: { timestamp },
+    auditbeatAlias,
+    logAlias,
+    packetbeatAlias,
+  },
+}: RequestBasicOptions) => {
+  const filter = [
+    ...createQueryFilterClauses(filterQuery),
+    {
+      range: {
+        [timestamp]: {
+          gte: from,
+          lte: to,
+        },
+      },
+    },
+  ];
+
+  const dslQuery = {
```

Comment:
plz extra double check these!

Reply:
Thanks for the note and letting people know ahead of time what to focus on due to uncertainties. I will do some queries below to see some of the differences and pointers best I can help out.